### PR TITLE
Added Slides for october meetup

### PR DESCRIPTION
**Pull request by zf2timo**

### CODE REVIEW

File: phpugms_greeting_2018-10/index.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+        <section>
+            <h1>Upcoming Events</h1>
+        </section>
+
+
+        <section class="l-2up">
+            <h2>November Meeting</h2>
+
+            <strong>Tuesday, November 20, 2018</strong><br>
```
Comment (bcremer) [inline phpugms_greeting_2018-10/index.html:117]: Location CHECK24 Münster

Comment (bcremer) [reply phpugms_greeting_2018-10/index.html:117]: Maybe we can also announce our End-Of-Year meetup already since location and date is already fix.

Comment (zf2timo) [reply phpugms_greeting_2018-10/index.html:117]: all done.

**Comment by bcremer** [comment]
Looks fine for me. Thanks for preparing. Please merge whenever you are ready.

**Pull request merged**